tools/check_fs_content.py

```diff
@@ -159,7 +159,7 @@ def print_report(broken, empty, missing, reboot):
 
 # Relative path from LekaOS root
 ROOT_PATH = args.dir if not args.test else 'fs/usr/test'
-LOOP_SLEEP_DELAY = args.SLEEP_DELAY if not args.FAST else 0.2
+LOOP_SLEEP_DELAY = args.loop_delay if not args.fast else 0.2
```

Comment:
Just wondering why `metavar` cannot be used, but it's only used on help command so that is fine [[Link](https://docs.python.org/3/library/argparse.html#metavar)]